ios/brave-ios/Sources/Brave/Extensions/ContinuousClock+Extensions.swift

```diff
@@ -0,0 +1,29 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+extension ContinuousClock.Instant {
```

Comment:
Do we want this clock to run in prod?

Reply:
Not sure, these debugs are really useful to know how much adblock launching affects the app. If I make a change. If it doesn't harm anything I would like to keep it